include/mega/node.h

```diff
@@ -67,8 +69,16 @@ struct MEGA_API NewNode : public NodeCore
 
     bool added;
 
+    MEGA_DISABLE_COPY(NewNode)
```

Comment:
no need to disable copy here; its base class has it disabled so it's disabled here too unless you write one yourself